Exadel.Compreface.UnitTests/Services/SubjectServiceTests.cs

```diff
@@ -18,8 +19,9 @@ public class SubjectServiceTests
         public SubjectServiceTests()
         {
             var apiKey = GetRandomString();
-            var baseUrl = GetRandomString();
-            var configuration = new ComprefaceConfiguration(apiKey, baseUrl);
+            var domain = GetRandomString();
+            var port = new Random().Next().ToString();
+            var configuration = new ComprefaceConfiguration(apiKey, domain, port);
```

Comment:
Don't repeat the same code, better add it to separate method. You can look RecognitionServiceTests

Reply:
@Ukrainiangirl, Do you mean that each class with service tests has a couple of lines of the same code with ComprefaceConfiguration initialization in the constructor? and we should take out it in some base class? just to avoid code duplication?